# HarvestSchedule.srv
string robot_id
float32 crop_yield
string status
---
string schedule
